modules/Microsoft.KeyVault/vaults/version.json

```diff
@@ -1,4 +1,4 @@
 {
     "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-    "version": "0.5"
+    "version": "0.6"
```

Comment:
Is this bump intended to test the correct publishing or should it be reverted?

Reply:
should be reverted. Good catch

Reply:
Fixed